settings/OpeningPeriodForm/InputFields.js

```diff
@@ -71,6 +71,9 @@ class InputFields extends React.Component {
           label={<FormattedMessage id="ui-calendar.validFrom" />}
           onChange={this.setStartDate}
           required
+          timeZone="UTC"
+          backendDateStandard="YYYY-MM-DD"
+          dateFormat="MM/DD/YYYY"
```

Comment:
I confess I'm not familiar with this module, but I'm surprised to see a locale-specific timezone, `MM/DD/YYYY`, here. Are you certain hard-coding a specific date format is the correct solution here?

Reply:
I've just used format used among the whole module, but without receiving it from translations.

[Component](https://github.com/folio-org/ui-calendar/blob/cac1ee26da4745b4d38bbb1a4e4ec183bed81e06/settings/ServicePointDetails.js#L105)

[Translations](https://github.com/folio-org/ui-calendar/blob/cac1ee26da4745b4d38bbb1a4e4ec183bed81e06/translations/ui-calendar/en.json#L72)

Reply:
Isn't date-format is locale-dependent? i.e. in the en-US locale people expect to see mm/dd/yyyy, in sp they expect dd/mm/yyyy, in de dd.mm.yyyy etc.